generate.sh

```diff
@@ -139,13 +140,32 @@ generateJavascript() {
   return 0
 }
 
+generatePython() {
+  LIBRARY="$1"
+  OPERATION="$2"
+  ARTIFACT_ID="symbol-openapi-$LIBRARY-client"
+  echo "Generating $LIBRARY"
+  rm -rf "$BUILD_DIR/$ARTIFACT_ID"
+  openapi-generator generate -g "$LIBRARY" \
+    -o "$BUILD_DIR/$ARTIFACT_ID" \
+    -t "$LIBRARY-templates/" \
+    -i "$INPUT" \
+    --additional-properties="projectName=$ARTIFACT_ID" \
+    --additional-properties="packageVersion=$VERSION" \
+    --additional-properties="snapshot=$SNAPSHOT" \
+    --type-mappings=x-number-string=int
+  echo "DONE"
+  return 0
```

Comment:
Try to at least compile the generated code to validate if it's fine. Are you planning to target the lib to multiple python versions? Maybe requiring python 3.7+ is too restrictive?

There is a generated .travis file in the `build/symbol-openapi-python-client` folder. It can give you a hint about how to compile it.

Reply:
Python code is validated by running tests. The openapi-generator generates a test suite to validate the generated code and all tests pass.

I've lowered the requirement to python 3.4+. However, the recommended is 3.7+ for compatibility with Python Catbuffer library. Also note that Python 3.4 has already reached [End Of Life](https://devguide.python.org/#status-of-python-branches).

I've included a .travis file but as mentioned earlier will be working on the release/publish as a separate task #28